ineffassign.go

```diff
@@ -37,7 +37,9 @@ func main() {
 			return nil
 		}
 		if fi.IsDir() {
-			if *dontRecurseFlag && path != root || filepath.Base(path) == "testdata" {
+			if *dontRecurseFlag && path != root ||
+				filepath.Base(path) == "testdata" ||
+				strings.Contains(path, "/vendor/") {
```

Comment:
I think this should read
```
if path != root && (*dontRecurseFlag || filepath.Base(path) == "testdata" || filepath.Base(path) == "vendor") {
```
to avoid recursing into such directories but to allow the tool to run directly within them, which users could conceivably desire.